app/src/app/components/files/files.component.html

```diff
@@ -40,7 +40,8 @@
         >
           <data-columns>
             <ng-container *ngFor="let column of columns; trackBy: trackByColumnId">
-              <ng-container *ngIf="column.template && !(column.desktopOnly && isSmallScreen)">
+              <ng-container *ngIf="column.template && !(column.desktopOnly && isSmallScreen) &&
```

Comment:
this is not going to work for ACA, because this is a template for the ngFor and it is going to hide all columns that are not "app.files.name" and other stuff... also, the ACA should not be relying on the "securityMarks" as it's the feature from another application